hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -2777,7 +2777,7 @@
 
             - Provision an RMA_UNLOCK token in OTP.
             - Reset the chip.
-            - Ensure device id and manufacturing state can be read through the LC JTAG.
+            - Ensure chip revision, device id and manufacturing state can be read through the LC JTAG.
```

Comment:
I would probably also add resetting the chip a few times and randomize these entries in OTP on each reset, so that all bits show up adequately as covered.

Reply:
yes good point, let me add that.